rts/lra/lra-client/src/main/java/io/narayana/lra/client/LRAInfoImpl.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
How come the implementation class has this licence - can't it be our normal licence?